src/styles/App.scss

```diff
@@ -142,6 +157,44 @@ $dragBarWidth: 10px;
 
 
 
+#tabpages.full-width {
+	top: 60px;
+	/* 调整这个值以匹配新的 header 高度 */
+	left: 0;
+	width: calc(100% - 300px);
+	/* 假设右侧面板宽度为 300px，根据实际情况调整 */
```

Comment:
Translations:

/* Adjust this value to match the new header height */
left: 0;
width: calc(100% - 300px);
/* Assume that the width of the right panel is 300px, adjust it according to the actual situation */